recipes/inspec_package.rb

```diff
@@ -0,0 +1,28 @@
+# encoding: utf-8
+#
+# Cookbook Name:: audit
+# Recipe:: inspec_package
+#
+# Copyright 2016 Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+chef_ingredient 'inspec' do
+  version node['audit']['inspec_version'] if node['audit']['inspec_version'] != 'latest'
+  accept_license true
+  action :upgrade
+end
+
+load_inspec_libs
```

Comment:
oh shoot i think i had some leftover gem cruft in there when i tested.  right, ok, so we would need to set the load path like you did there. sounds like there are some other issues too (looking in compliance-support) in that chef-ingredient uses mixlib install gem.  so we could go the remote file and execute route. and then there's the problem of having gem and package installed....we could do a check to get around that i suppose